ui/src/components/ui/screen-quality-selector/screen-quality-selector.tsx

```diff
@@ -0,0 +1,63 @@
+import { useEffect } from 'react'
+import { observer } from 'mobx-react-lite'
+import { useParams } from 'react-router-dom'
+import { useTranslation } from 'react-i18next'
+import { Flex, Caption, Button } from '@vkontakte/vkui'
+import { Icon16Connection } from '@vkontakte/icons'
+
+import { MarkedSliderRange } from '@/components/lib/marked-slider-range'
+import { PopoverContainer } from '@/components/lib/popover-container'
+
+import { useDebounce } from '@/lib/hooks/use-debounce.hook'
+import { deviceControlStore } from '@/store/device-control-store'
+
+import styles from './screen-quality-selector.module.css'
+
+const QUALITY_OPTIONS = [10, 20, 30, 40, 50, 60, 70, 80]
+
+export const ScreenQualitySelector = observer(() => {
+  const { t } = useTranslation()
+  const { serial } = useParams()
+  const debouncedQuality = useDebounce(deviceControlStore.currentQuality, 300)
```

Comment:
I think I can't debounce setCurrentQuality because it would cause the slider state to update with a delay, making it difficult for the user to interact with the slider smoothly.